docs/src/pages/[platform]/connected-components/liveness/intro.react.mdx

```diff
@@ -1,6 +1,3 @@
-import { Example } from '@/components/Example';
-import { LivenessQuickStartReact } from './QuickStartReact';
-
-<Example>
-  <LivenessQuickStartReact />
-</Example>
+<video controls width="100%" style={{maxWidth: '320px'}}>
```

Comment:
Should we also remove the `LivenessQuickStartReact` component if not used anymore? Or if used remove the THmeProvider wrapper from it.

Reply:
Removed in latest commit, thanks!